src/components/Header/UserMenu.tsx

```diff
@@ -0,0 +1,78 @@
+import { Fragment } from 'react';
+import { HiUser } from 'react-icons/hi';
+import { Menu, Transition } from '@headlessui/react';
+import { type User } from '@supabase/gotrue-js/src/lib/types';
+import cn from 'classnames';
+
+import { useSignOut } from './hooks';
```

Comment:
Remove the empty line
```suggestion
import cn from 'classnames';
import { useSignOut } from './hooks';
```

Reply:
I prefer a clear distinction between libraries and local file imports, but whatever, not a big deal.